src/lib/api.js

```diff
@@ -49,7 +50,9 @@ export default async function API(): Promise<ApolloClient> {
 		}
 	} );
 
-	const httpLink = new HttpLink( { uri: API_URL, headers: headers } );
+	const httpLink = new HttpLink( { uri: API_URL, headers: headers, fetchOptions: {
+		agent: process.env.hasOwnProperty( 'VIP_PROXY' ) ? ProxyAgent( process.env.VIP_PROXY ) : null,
```

Comment:
Should this be `new ProxyAgent()`?
```suggestion
		agent: process.env.hasOwnProperty( 'VIP_PROXY' ) ? new ProxyAgent( process.env.VIP_PROXY ) : null,
```

Reply:
It should indeed.